.github/workflows/push.yaml

```diff
@@ -54,6 +54,15 @@ jobs:
           DOCKER_HUB_USERNAME: ${{ secrets.DOCKER_HUB_USERNAME }}
         run: bin/build-browser-tests
 
+      - name: Run build-localstack-env.
```

Comment:
I'm happy to make this change separately if preferred.

Reply:
Can we rename all of these "build and push"?

Reply:
The script name or the GitHub action name?

I went ahead and renamed this GitHub action and the `build_browser_tests` GitHub action. I hesitate to update the script names since they're really build and optionally push. Also would prefer to keep their current names for shorter file names.

Reply:
Just the "name:" part in the github action